perftests/Xcode/PerfTests/NinjaPerfTests.mm

```diff
@@ -30,6 +30,15 @@ static void ExecuteShellCommand(const char *String) {
 
 @implementation NinjaPerfTests
 
++ (NSString*) findNinja {
+    NSString* overrideNinjaPath = [@(SRCROOT)
+                                   stringByAppendingPathComponent:@"llbuild-test-tools/utils/Xcode/ninja"];
```

Comment:
Is this even really needed?

Reply:
I expect that the perf test bots are set up this way.  My goal here was to not change the existing behavior for those that have it set up as such, while adding a sane default.  You could conceivably want to override what version of ninja you want to compare against.